### PR TITLE
Fix back skip

### PR DESCRIPTION
**Pull request by parkouss**

### CODE REVIEW

File: mozregression/history.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    """
+    An history bisection step.
+    """
+    def __init__(self, build_range, index, verdict=None):
+        self.build_range = build_range
+        self.index = index
+        self.verdict = verdict
+
+
+class BisectionHistory(object):
```
Comment (wlach) [inline mozregression/history.py]: You might consider making this class inherit from list, and just adding one or two custom methods for convenience. Something like:

```
class BisectionHistory(list):
    def add_step(self, build_range, index, verdict):
        self.append(BisectionStep(...))
```

File: mozregression/history.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+"""
+Representation of the bisection history.
+"""
+
+
+class BisectionStep(object):
```
Comment (wlach) [inline mozregression/history.py]: Could this be a namedtuple? https://docs.python.org/2.7/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields

Comment (parkouss) [inline mozregression/history.py]: Sure, good idea.

File: mozregression/history.py (hunk 1 above)
Comment (parkouss) [inline mozregression/history.py]: Will do!

**Pull request merged**